eq-author/src/App/review/ReviewPage.js

```diff
@@ -61,23 +63,57 @@ const ReviewPage = ({ match, history }) => {
     return <Redirect to={`/q/${match.params.questionnaireId}`} />;
   }
 
+  const [comment, setComment] = useState({ value: "" });
+
   return (
     <Container>
       <Header title="Review" />
       <ContentContainer>
         <Label>Questionnaire review</Label>
-        <Caption>
-          This questionnaire is awaiting review, please approve or reject below
-        </Caption>
+        <Caption>This questionnaire is awaiting review.</Caption>
         <Separator />
+        <InformationPanel>
+          Provide a comment if you are rejecting. Your comment will be displayed
+          to the questionnaire editors in the History page.
+        </InformationPanel>
+        <RichTextEditor
+          id={`reject-reason-textarea`}
+          name="rejectReason"
+          label="Reason for rejecting"
+          controls={{
+            emphasis: true,
```

Comment:
Noticed that the Heading 'H' is available in the design but not in your version
use 
heading: true,
in the controls